orm/encoding/ormkv/entry.go

```diff
@@ -45,7 +45,12 @@ func (p *PrimaryKeyEntry) String() string {
 	if p.Value == nil {
 		return fmt.Sprintf("PK %s %s -> _", p.TableName, fmtValues(p.Key))
 	} else {
-		return fmt.Sprintf("PK %s %s -> %s", p.TableName, fmtValues(p.Key), p.Value)
+		valBz, err := stablejson.Marshal(p.Value)
+		valStr := string(valBz)
+		if err != nil {
+			valStr = fmt.Sprintf("ERR %v", err)
+		}
+		return fmt.Sprintf("PK %s %s -> %s", p.TableName, fmtValues(p.Key), valStr)
```

Comment:
It looks like this change has caused a slight change to the format of the entry and `entry_test.go` needs to be updated to reflect those changes (`i32:-1` becomes `{"i32":-1}`).

Reply:
thx! fixed